### DISCUSSION TITLE
pybricks-projects maintaince

### DISCUSSION THREAD

**Discussion by bochap** [pybricks/support#1402] [General]
Hi,

I was checking out the pybricks-projects and it looks like there are Open PRs from some time back. There are also some projects like https://github.com/pybricks/pybricks-projects/tree/master/sets/spike-prime/core-set/cnc_machine and https://github.com/pybricks/pybricks-projects/tree/master/sets/spike-prime/core-set/kiki_dog which do not show up on the documentation site. Is that repo still being maintained? Just checking in since the team has been great but there is so much for you all to maintain. I am working with my kid to work on some of the Spike examples on Lego Education using Pybricks. So wondering if I should put up something on the repo.

**Comment by laurensvalk** [comment]
Thanks for asking! We do still update the projects (see e.g. recently merged pull requests), but have probably unintentionally prioritized those that were (almost) ready to merge. It will certainly be good to give this a closer look when we get around to posting some samples with the new block code. Thanks for the reminder :-)
